README.md

```diff
@@ -1,7 +1,9 @@
-# faucet-pipeline
-[![npm](https://img.shields.io/npm/v/faucet-pipeline.svg)](https://www.npmjs.com/package/faucet-pipeline)
-[![Build Status](https://travis-ci.org/faucet-pipeline/faucet-pipeline.svg?branch=master)](https://travis-ci.org/faucet-pipeline/faucet-pipeline)
-[![Greenkeeper badge](https://badges.greenkeeper.io/faucet-pipeline/faucet-pipeline.svg)](https://greenkeeper.io)
+faucet-pipeline-core
+====================
+
+[![npm](https://img.shields.io/npm/v/faucet-pipeline-core.svg)](https://www.npmjs.com/package/faucet-pipeline-core)
+[![build status](https://travis-ci.org/faucet-pipeline/faucet-pipeline-core.svg?branch=master)](https://travis-ci.org/faucet-pipeline/faucet-pipeline-core)
+[![Greenkeeper badge](https://badges.greenkeeper.io/faucet-pipeline/faucet-pipeline-core.svg)](https://greenkeeper.io)
```

Comment:
Not sure whether we need to explicitly update any of these services. 🤷‍♂️

Reply:
Sure. All of them.